### PR TITLE
Fix type error in inviteToTeam

### PR DESCRIPTION
**Pull request by Dschoordsch**
primisify returns any, thus a typo in `toString` was not caught.

Resolves: #5912

### CODE REVIEW

**Review by nickoferrall — APPROVED** [review]
Nice catch, LGTM!

**Review by BartoszJarocki — APPROVED** [review]

**Comment by nickoferrall** [comment]
Skipped Maintainer Review and merged as it's been approved by two reviewers and it's holding up the ship

**Pull request merged**
